es_ES.json

```diff
@@ -0,0 +1,798 @@
+    "CREATOR_STORAGE_CANNOT_OBTAIN_JSON": "No se pueden obtener datos JSON",
+    "PERSIST_CAR_TITLE": "Persistir Coche",
+    "PERSIST_CAR_INVALID_VEHICLE_SAVE_ATTEMPT": "Intentado salvar un vehículo que no existe",
+    "TELEPORT": "Teletransporte",
+    "TELEPORT_PLAYER_IS_DEAD": "El jugador objetivo está muerto.",
+    "TELEPORT_PLAYER_IS_NOT_IN_VEHICLE": "El jugador objetivo no está en un vehículo.",
+    "TELEPORT_FAILED_TO_TAKE_CONTROL": "No se ha podido tomar el control del vehículo del jugador.",
+    "TELEPORT_INVALID_VEHICLE": "No se ha podido encontrar ningún vehículo válido.",
+    "TELEPORT_NO_SEATS_FREE": "No hay asientos libres en el vehículo del jugador.",
+    "TELEPORT_NO_WAYPOINT_SET": "Error al encontrar la posición del waypoint.",
+    "TELEPORT_NO_OBJECTIVE": "No ha encontrado una posición objetiva",
+    "MODIFY_TIME": "Modificar la hora",
+    "WARP_TIME_TITLE": "Tiempo de deformación",
+    "MODIFY_TIME_HOST_REQUIRED": "Para modificar la hora se necesita el host de sesión",
+    "MODIFY_TIME_NO_PLAYER_TIMESTAMP": "Todavía no tenemos la marca de tiempo del jugador",
+    "HIJACK_TRAIN_FOUND_TRAIN": "Encontré un tren cerca",
+    "HIJACK_TRAIN_DELETED_TRAIN": "Eliminado el tren cercano",
+    "VEHICLE_INVALID": "El vehículo no es válido.",
+    "VEHICLE_FAILED_CONTROL": "Fallo al tomar el control del vehículo a distancia.",
+    "REMOTE_CONTROL": "Mando a distancia",
+    "KICKED": "Pateado",
+    "USER_DESYNC_KICKED": "Has sido desincronizado.",
+    "PLAYER_JOINED": "Jugador Incorporado",
+    "PLAYER_LEFT": "Jugador Izquierda",
+    "PLAYER_LEFT_INFO": "{} ha liberado la ranura {} con el ID de Rockstar: {}",
+    "PLAYER_JOINED_INFO": "{} ha ocupado la ranura {} con el ID de Rockstar: {}",
+    "PLAYER_CHANGED_NAME": "{} cambió su nombre a {}",
+    "NETWORK_PLAYER_MGR": "Gestor de reproductores en red",
+    "NETWORK_PLAYER_MGR_INIT": "Entrar en sesión e inicializar los datos del jugador.",
+    "NETWORK_PLAYER_MGR_DESTROY": "Abandonar la sesión y limpiar los datos del jugador.",
+    "BLOCK_JOIN_INFO": "{} ha sido bloqueado al intentar unirse",
+    "PROTECTIONS": "Protecciones",
+    "REMOTE_CRASH_INVALID_TRANSITION_FROM": "Bloqueada la transición inválida de crasheo de {}",
+    "REMOTE_CRASH_INVALID_TRANSITION": "Bloqueo de transición no válida crasheo remoto",
+    "REMOTE_CRASH_INVALID_BLACKLIST_FROM": "Bloqueado fallo de lista negra inválida de {}",
+    "REMOTE_CRASH_INVALID_BLACKLIST": "Bloqueo de listas negras inválidas",
+    "OOM_KICK": "{} ha intentado expulsarte!",
+    "BLOCK_JOIN_PREVENT_PLAYER_JOIN": "Intentando evitar que {} se una...",
+    "REMOTE_KICK_LOST_CONNECTION": "¡Alguien intentó perder la conexión expulsandote remotamente!",
+    "REMOTE_KICK_LOST_CONNECTION_PLAYER": "¡Alguien intentó expulsar a {} remotamente!",
+    "REMOTE_KICK_BREAKUP": "¡Alguien ha intentado expulsarte a distancia!",
+    "REMOTE_KICK_BREAKUP_PLAYER": "¡Alguien ha intentado de expulsar a otro jugador a distancia!",
```

Comment:
English: `Someone tried to breakup kick {} remotely!`

Your translation is missing the `{}` (which will be the name of the player trying to kick you)